packages/website/src/components/Schedule/ScheduleItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Center } from '@coderscamp/ui/components/Center';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+interface ScheduleItemProps {
+  index: number;
+  date: string;
+  description: string;
+}
+
+export const ScheduleItem = ({ index, date, description }: ScheduleItemProps) => {
+  return (
+    <Center
+      flexDirection="column"
+      justifyContent="start"
+      textAlign="center"
+      w="min(400px, 95%)"
+      h="292px"
```

Comment:
🤔 The smallest viewport size we need to consider is 320px. On such size this implementation of card brokes.
<img width="312" alt="CleanShot 2021-08-11 at 22 24 09@2x" src="https://user-images.githubusercontent.com/22938071/129098147-ccb9e075-d4c3-4480-a51c-b15ea8c5b594.png">
As a solution, you can use `minH` instead of height here and let the padding do the job.